src/Model/batchprocessing/BatchProcessKaplanMeier.py

```diff
@@ -0,0 +1,163 @@
+import kaplanmeier as km
+import matplotlib as mp1
+mp1.use("TkAgg")
```

Comment:
Tkinter isnt included as a dependency? This would cause the PR not to build if it isn't installed